classroom-dashboard/app/assets/stylesheets/students.scss

```diff
@@ -3,9 +3,4 @@
 // You can use Sass (SCSS) here: http://sass-lang.com/
 @import "bootstrap-sprockets";
 @import "bootstrap";
-
-.student {
-    font-family: 'Work Sans', sans-serif;
-    height: 350px;
-    width: 250px;
-}
+@import "*";
```

Comment:
I'm not sure what this import statement does. From my understanding normally ```*``` refers to the regex catch-all character, but I'd think this would import the entire internet or something. Does this refer to a specific gem or something along those lines?